dvc/tree/base.py

```diff
@@ -59,6 +59,7 @@ class BaseTree:
     TRAVERSE_PREFIX_LEN = 3
     TRAVERSE_THRESHOLD_SIZE = 500000
     CAN_TRAVERSE = True
+    CHUNK_SIZE = 64 * 1024 * 1024  # 64 MiB
```

Comment:
Could you elaborate why this particular chunk size?

Reply:
In general, the job count is 16 for a normal computer (4 cores), so I wanted to fix it to 1 GiB at most (`64 * 16 = 1024`). For very good end computers (like 8 cores or something like that) it will grow up gradually to 2GB (and 4GB, ...). Making it 128 is also an option, but 64 MiB per chunk seems not that bad (256 MiB is I think very huge, at least that is what my previous experience with it showed).